packages/core/src/components/utils/theme-switcher.ts

```diff
@@ -47,11 +47,15 @@ class ThemeSwitcher {
     );
   }
 
-  public setTheme(themeName: string, systemAppearance = false) {
+  public setTheme(
+    themeName: string,
+    systemAppearance = false,
+    target = document.body
```

Comment:
Can we add here a "todo" with an new jira ticket, that we will replace it with document in v4?